.github/workflows/pr-tests-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Run documentation tests
```

Comment:
I still don't know how the documentation is generated. I'm just wondering what is meant by the word `tests` here? Just checking that the specified commands fail/success? At the same time, the documentation is not published anywhere?

Reply:
Right now, it just checks that the documentation is capable of being built.  There are no 'tests', per se.  You are correct that the documentation does not get published anywhere in this workflow.

Reply:
Ok